.github/workflows/bump-sec-scanners-config-reusable.yml

```diff
@@ -157,11 +156,11 @@ jobs:
         run: |
           echo "please review  ${PR_URL}"
 
-      - name: Wait for PR to be Merged
         if: ${{ env.CREATE_PR == 'true' }}
```

Comment:
Have you tested this line if it worked as before? I think, it doesn't work as if is outside the job.

Reply:
`echo "PR_URL=${PR_URL}" >> $GITHUB_ENV`
is like defining a env var in the job; it will become available in all the steps.